.gitignore

```diff
@@ -13,3 +13,4 @@ results
 
 npm-debug.log
 node_modules
+.idea
```

Comment:
This shouldn't be here, you have to manage a global ignore file into your own environment.
You can find many articles about this fact on internet like this one :
https://blog.martinhujer.cz/dont-put-idea-vscode-directories-to-projects-gitignore/